performance_manager/lib/postgres_utils.py

```diff
@@ -1,20 +1,23 @@
 import logging
 import os
+import json
 import pathlib
 import urllib.parse
-
-from typing import Dict, List
+from typing import Dict, List, Any
+import pandas
 
 import sqlalchemy as sa
-from sqlalchemy.engine import URL as DbUrl  # type: ignore
```

Comment:
`sqlalchemy2-stubs` lets us get rid of all these `# type: ignore` 's

Reply:
🙏